src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -99,24 +101,25 @@ public ArmSubsystem() {
   public void setPosition(double angleDegrees) {
     targetDegrees = angleDegrees;
     armMotor.setControl(
-        new MotionMagicVoltage(Constants.Arm.ANGLE_TO_ENCODER_ROTATIONS(angleDegrees)));
+        controlRequest.withPosition(Constants.Arm.ANGLE_TO_ENCODER_ROTATIONS(angleDegrees)));
   }
 
-  private double getDegrees() {
+  private double calculateDegrees() {
```

Comment:
On 581 we like to use "calculate" to describe a potentially expensive computation that should only be used once per robot loop. To access the previously computed value, we use "get" in method names for the cheaper counterparts which just retrieve the data.